src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,201 @@
+}
+.Navigation .NavLinks .Showing{
+  display: inline-block;
+}
+.Navigation .UserProfileName{
+  /* display: inline; */
+  padding: 1.3rem 1rem;
+  /* padding-top: 90px; */
+}
+.Navigation .UserProfileName strong{
+  padding-bottom: 4rem;
+  color: black;
+}
+.UserProfileName .NameUser{
+  display: inherit;
+}
+.Navigation i {
+  border: solid black;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  margin-top: 2rem;
+  padding: 5px;
+  transform: rotate(45deg);
+  /* -webkit-transform: rotate(45deg); */
+}
+
+.Navigation > .NavLinks > a:hover {
+  height: 100%;
+  font-weight: bold;
+}
+.Navigation > .NavLinks a:last-child {
+  padding-right: 160px;
+}
+.Navigation > #nav-check {
+  display: none;
+}
+.Navigation .Nav-dropdown{
+  height: 100%;
+  position: relative;
+  display: inline-block;
+}
+.Navigation .Nav-dropdown select{
+  float: right;
+  height: 70%;
+  margin-top: 15px;
+  border-radius: 6px;
+  width: 130px;
+  padding-left: 5px;
+}
+.Navigation select option{
+  font-size: 16px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+.Navigation .DropdownIcon img{
+  width: 50px;
+  vertical-align: middle;
+  border-style: none;
+  box-sizing: none;
+}
+
+.Navigation .Nav-dropdown .DropdownIcon{
+  /* font-size: 16px;   */
+  /* float: left; */
+  border: none;
+  outline: none;
+  color: white;
+  padding: 1px 16px;
+  background-color: inherit;
+  font-family: inherit;
+  margin: 0;
+}
+
+.Navigation .DropdownContent{
+  display: none;
+  position: absolute;
+  background-color: #f9f9f9;
+  min-width: 260px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+}
+.Navigation .DropdownContent a{
+  /* float: none; */
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+  padding: 15px 5px;
+  text-align: center;
+}
+.Navigation .DropdownContent a:hover{
+  background-color: #ddd;
+}
+.Navigation .Nav-dropdown:hover .DropdownContent{
+  display: block;
+}
+
+@media (max-width:600px) {
+  .Navigation > .Nav-btn {
+    display: inline-block;
+    position: absolute;
+    right: 0px;
+    top: 0px;
+  }
+  .Navigation > .Nav-btn > label {
+    display: inline-block;
+    width: 50px;
+    height: 50px;
+    padding: 13px;
+  }
+  .Navigation > .Nav-btn > label:hover,.Navigation  #nav-check:checked ~ .Nav-btn > label {
+    background-color: rgba(0, 0, 0, 0.3);
+  }
+  .Navigation > .Nav-btn > label > span {
+    display: block;
+    width: 25px;
+    height: 10px;
+    border-top: 2px solid #eee;
+  }
+  .Navigation > .Nav-links {
+    position: absolute;
+    display: block;
+    width: 100%;
+    background-color: #333;
+    height: 0px;
+    transition: all 0.3s ease-in;
+    overflow-y: hidden;
+    top: 50px;
+    left: 0px;
+  }
+  .Navigation > .Nav-links > Link {
+    display: block;
+    width: 100%;
+  }
+  .Navigation > #nav-check:not(:checked) ~ .Nav-links {
+    height: 0px;
+  }
+  .Navigation > #nav-check:checked ~ .Nav-links {
+    height: calc(100vh - 50px);
```

Comment:
`Nav-links` and `Nav-btn` these are not called anywhere

Reply:
@Mubangizi this has been fixed